src/v3/types.ts

```diff
@@ -28,13 +28,25 @@ export interface StreamPublishMetadata {
   alias?: string;
 }
 
+export interface DataChannelMetadata {
+  endpointId?: string;
+  label: string;
+  streamId: number;
+  participantId?: string;
```

Comment:
If this is analogous to the new `StreamPublishMetadata`, does it need `participantId` or `endpointId` at all?